toolchain-msvc.cmake

```diff
@@ -1,13 +1,6 @@
 
-if(NOT ARCH)
-    set(ARCH i386)
-endif()
-
-# Default to Debug for the build type
```

Comment:
`msvc`: Still defaults to `Debug`, somehow :-)
`msbuild`: Now 'empty' instead of `Debug` :-/